src/main/java/com/github/jchartier/slackbot/pomodoro/PomodoroCommandManager.java

```diff
@@ -49,6 +56,12 @@ public void register() {
                 return;
             }
 
+            if (isListCommand(message)) {
```

Comment:
Even a switch case is not the cleanest way to do it. We could use a factory and/or command pattern to nicely manage all possible commands (https://github.com/joelchartier/slackbot-pomodoro/issues/11)